metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -1673,7 +1595,30 @@ def _get_upstream_lineage(
                                 make_schema_field_urn(
                                     upstream_dataset_urn, upstream_field
                                 )
-                                for upstream_field in field.upstream_fields
+                                for upstream_field in cast(
+                                    List[str], field.upstream_fields
```

Comment:
I updated the ViewField.upstream_fields from List[str] to `upstream_fields: Union[List[str], List[ColumnRef]]`. Because it is either a column name in-case of non-sql lineage or ColumnRef for sql query parsed result

Reply:
why can't it always be `List[ColumnRef]`? we can convert the strings to ColumnRef so that it's all uniform